osu.Game/Overlays/NewsOverlay.cs

```diff
@@ -56,13 +57,31 @@ private void load(OsuColour colours)
             };
 
             header.Current.BindTo(Current);
+
             Current.TriggerChange();
         }
 
-        public void ShowFrontPage()
+        private CancellationTokenSource loadChildCancellation;
+
+        protected void LoadChildContent(NewsContent newContent)
         {
-            Current.Value = null;
+            content.FadeTo(0.2f, 300, Easing.OutQuint);
+
+            loadChildCancellation?.Cancel();
+
+            LoadComponentAsync(newContent, c =>
+            {
+                content.Child = c;
+                content.FadeIn(300, Easing.OutQuint);
+            }, (loadChildCancellation = new CancellationTokenSource()).Token);
+        }
+
+        public void ShowArticle(string article)
```

Comment:
this isn't being used anywhere, right? probably best save it for the PR where it's being used.